test/features/cr/deploy_quarkus_service.feature

```diff
@@ -56,4 +56,34 @@ Feature: CR: Deploy quarkus service
     @native
     Examples:
       | native | minutes |
-      | "enabled" | 20 |
\ No newline at end of file
+      | "enabled" | 20 |
+
+#####
+
+  # Disabled as long as https://issues.redhat.com/browse/KOGITO-1163 and https://issues.redhat.com/browse/KOGITO-1166 is not solved
```

Comment:
Do we have a follow-up task on those issues to enable the test when they are solved ? :)

Reply:
Not yet, will create one.

Reply:
Created https://issues.redhat.com/browse/KOGITO-1265